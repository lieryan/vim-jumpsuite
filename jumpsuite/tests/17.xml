<?xml version="1.0" encoding="utf-8"?><testsuites><testsuite name="pytest" errors="1" failures="0" skipped="0" tests="4" time="21.079" timestamp="2022-10-04T19:13:30.906087" hostname="ryan-delta"><testcase classname="tests.test_template_api" name="test_dependency[api]" file="tests/test_template_api.py" line="48" time="0.069" /><testcase classname="tests.test_template_api.TestDocker" name="test_docker_build_is_successful[0-api-0]" file="tests/test_template_api.py" line="54" time="19.061" /><testcase classname="tests.test_template_api.TestDocker" name="test_docker_start_app_application_is_listening_on_port_80[0-api-0]" file="tests/test_template_api.py" line="57" time="1.935"><error message="failed on teardown with &quot;subprocess.CalledProcessError: Command '['docker', 'image', 'rm', 'cookiecutter-firmus-unittest:97f30a51dee8f710c16f972eb9f9c21b696f6e9b']' returned non-zero exit status 1.&quot;">Traceback (most recent call last):
  File "/home/lieryan/Projects/cookiecutter-firmus/tests/test_template_api.py", line 107, in docker_build
    proc.check_returncode()
  File "/usr/lib/python3.10/subprocess.py", line 456, in check_returncode
    raise CalledProcessError(self.returncode, self.args, self.stdout,
subprocess.CalledProcessError: Command '['docker', 'image', 'rm', 'cookiecutter-firmus-unittest:97f30a51dee8f710c16f972eb9f9c21b696f6e9b']' returned non-zero exit status 1.</error></testcase></testsuite></testsuites>