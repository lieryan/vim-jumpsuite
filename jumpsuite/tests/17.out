FAILED. Ran 4 tests in 21.079s (errors=1, failures=0, skipped=0)
/home/lieryan/Projects/cookiecutter-firmus/tests/test_template_api.py:107: docker_build : subprocess.CalledProcessError: Command '['docker', 'image', 'rm', 'cookiecutter-firmus-unittest:97f30a51dee8f710c16f972eb9f9c21b696f6e9b']' returned non-zero exit status 1. : proc.check_returncode()
/usr/lib/python3.10/subprocess.py:456:`-  check_returncode :  : raise CalledProcessError(self.returncode, self.args, self.stdout,
